addons/web_editor/static/src/js/backend/html_field.js

```diff
@@ -203,6 +203,7 @@ export class HtmlField extends Component {
                                             validateCallback: this.onDynamicPlaceholderValidate.bind(this),
                                             closeCallback: this.onDynamicPlaceholderClose.bind(this),
                                             positionCallback: this.positionDynamicPlaceholder.bind(this),
+                                            preventClose: () => document.activeElement.closest(".o_popover"),
```

Comment:
@visp-odoo `this.wysiwyg.$iframe[0].closest(".o_popover")` , we can use this instead of finding it directly from document.